impeller/entity/geometry/stroke_path_geometry.cc

```diff
@@ -234,7 +236,9 @@ StrokePathGeometry::CreateSolidStrokeVertices(
     const StrokePathGeometry::CapProc& cap_proc,
     Scalar scale) {
   VertexBufferBuilder<VS::PerVertexData> vtx_builder;
-  auto polyline = path.CreatePolyline(scale);
+  std::vector<Point> point_buffer;
+  point_buffer.reserve(512);
```

Comment:
document magic numbers? what is this from?

Reply:
I made it up! 

Shall I add a comment saying I made this up?

Reply:
Yes please!

Reply:
That way we remember its not a Chesterton's Fence situation in the future.

Reply:
Done